main.c

```diff
@@ -52,7 +52,8 @@ int main(int argc, char** argv) {
     printf("Failed to init SDL: %s\n", SDL_GetError());
     return 1;
   }
-  SDL_Window* window = SDL_CreateWindow("Zelda3", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, 512, 480, 0);

```

Comment:
This patch is not needed cause I added Alt+Enter for fullscreen.

Reply:
Okay I was not aware of that change. I'll remove it.